bedrock/firefox/redirects.py

```diff
@@ -42,7 +49,92 @@ def mobile_app(request, *args, **kwargs):
+    offsite_redirect(r"^firefox/faq/$", f"{FXC}/more/faq/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/$", f"{FXC}/features/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/adblocker/$", f"{FXC}/features/adblocker/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/add-ons/$", f"{FXC}/features/add-ons/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/block-fingerprinting/$", f"{FXC}/features/block-fingerprinting/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/bookmarks/$", f"{FXC}/features/bookmarks/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/customize/$", f"{FXC}/features/customize/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/eyedropper/$", f"{FXC}/features/eyedropper/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/fast/$", f"{FXC}/features/fast/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/password-manager/$", f"{FXC}/features/password-manager/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/pdf-editor/$", f"{FXC}/features/pdf-editor/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/picture-in-picture/$", f"{FXC}/features/picture-in-picture/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/pinned-tabs/$", f"{FXC}/features/pinned-tabs/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/private-browsing/$", f"{FXC}/features/private-browsing/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/private/$", f"{FXC}/features/private/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/sync/$", f"{FXC}/features/sync/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/tips/$", f"{FXC}/features/tips/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/features/translate/$", f"{FXC}/features/translate/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/installer-help/$", f"{FXC}/download/installer-help/", permanent=PERM_REDIRECTS),
+    # NOT YET - releasenotes and system requerements should be redirected as a separate piece of work
+    # offsite_redirect(r"^firefox/ios/releasenotes/$", f"{FXC}/firefox/ios/releasenotes/", permanent=PERM_REDIRECTS),
+    # offsite_redirect(r"^firefox/ios/system-requirements/$", f"{FXC}/firefox/ios/system-requirements/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/ios/testflight/$", f"{FXC}/channel/ios/testflight", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/linux/$", f"{FXC}/browsers/desktop/linux/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/mac/$", f"{FXC}/browsers/desktop/mac/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/mobile/get-app/$", f"{FXC}/browsers/mobile/get-app/", permanent=PERM_REDIRECTS),
```

Comment:
Dan asked this not redirect yet.

Reply:
It has no tests associated, so just:
```suggestion
    # NOT YET - let mobile redirectors attribute their respective sites
    # offsite_redirect(r"^firefox/mobile/get-app/$", f"{FXC}/browsers/mobile/get-app/", permanent=PERM_REDIRECTS),
```

Reply:
Circling back to this thread… it turns out neither of these two got redirected — so https://www.mozilla.org/firefox/mobile/get-app/ S2D form is still excluded from redirection, just to confirm it's intentional.